src/client/pages/RetroCertsLandingPage/index.js

```diff
@@ -26,11 +19,29 @@ function RetroCertsLandingPage(props) {
         <div className="container p-4">
           <h1>Hello</h1>
           <p>Weeks to certify: {userData.weeksToCertify.join(", ")}</p>
-          <p>
-            <Button variant="link" onClick={logout}>
-              Clear Session
-            </Button>
-          </p>
+          <Row>
+            <div className="col-md-4">
+              <Button
+                variant="outline-secondary"
+                className="text-dark bg-light"
+                href="/retroactive-certification/what-to-expect"
+              >
+                {t("retrocerts-weeks.button-back")}
+              </Button>
+            </div>
+            <div className="col-md-4">
+              <Button
+                variant="secondary"
+                href={
+                  "/retroactive-certification/certify/" +
```

Comment:
Can we use the path defined in data/routes here?

Reply:
Yeah, that's a good point. I'll fix that in a follow up change that also renames the landing page. routes.js isn't being used on the client side, but I think it should (instead of these hard coded paths). It also needs to be told about some of these new paths.